js/app.js

```diff
@@ -36,14 +36,14 @@ const domains_to_browser = {
   'bugs.webkit.org': 'WebKit'
 };
 
-const CLOSED_STATUSES = [
-  'FIXED',
-  'WONTFIX',
-  'CLOSED',
-  'RESOLVED',
-  'RESOLVED FIXED',
-  'DUPLICATE'
-];
+const STATUS = ['RESOLVED', 'VERIFIED', 'CLOSED'];
```

Comment:
This seems overly complicated for a few times. Let's just keep the array and add the extra item :)